kratos/utilities/function_parser_utility.cpp

```diff
@@ -141,8 +153,78 @@ void BasicGenericFunctionUtility::InitializeParser()
         const te_variable vars[] = {{"x", &x}, {"y", &y}, {"z", &z}, {"t", &t}, {"X", &X}, {"Y", &Y}, {"Z", &Z}};
 
         /* Compile the expression with variables. */
-        mpTinyExpr = te_compile(mFunctionBody.c_str(), vars, 7, &err);
-        KRATOS_ERROR_IF_NOT(mpTinyExpr) << "Parsing error in function: " << mFunctionBody << std::endl;
+        const bool c_like = StringUtilities::ContainsPartialString(mFunctionBody, "?") ? true : false;
```

Comment:
do we want to add support for this?

I feel like this is not necessary and with all the manual parsing it probably does more harm in the long run. So far the python syntax was sufficient, I don't think many ppl would use this

Reply:
No C like?

Reply:
As I said I would remove it